app/main.py

```diff
@@ -1,3 +1,15 @@
 def cache(func):
-    # Write your code here
-    pass
+    catche = {}
+
+    def inner(*args):
+
+        if args in catche:  # check previous results
+            print("Getting from cache")
+        else:
+            # if result not in catch,
+            # add result in cathc and return catch
```

Comment:
Here you have 3 different names: `cache`, `catche` and `catch`. What each of them means? :)